libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -222,6 +222,12 @@ class AP_BattMonitor
         return _params[instance]._serial_number;
     }
 
+#ifdef HAL_BUILD_AP_PERIPH
+    /// do_not_broadcast - returns true if battery instance is for INTERNAL USE ONLY and set to NOT broadcast CAN packets.
```

Comment:
Let's replace "Internal" with "only valid on AP_Periph".  I wonder if it is really a "broadcast".